### PR TITLE
Added size.md in the java folder

### LINKED ISSUE #1451: [Term Entry] Java Array: `.length` property
### Reason/inspiration (optional)

Add a new term entry under the [Java: Array](https://www.codecademy.com/resources/docs/java/array-list) concept. This entry should include:

- An explanation of the property.
- A "Syntax" section with a ```pseudo code snippet showing the general syntax.
- An "Example" section with a ```java code snippet showing how it works.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by dakshdeepHERE**
### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

Fixes #1465 #1450

### CODE REVIEW

File: content/java/concepts/array-list/terms/size/size.md
Comment (SSwiniarski) [inline content/java/concepts/array-list/terms/size/size.md:15]:
```suggestion
The **`size()`** method of the `ArrayList` class returns the number of elements in the list.
```

File: content/java/concepts/array-list/terms/size/size.md
Comment (SSwiniarski) [inline content/java/concepts/array-list/terms/size/size.md:20]:
```suggestion
int size = list.size()
```

File: content/java/concepts/array-list/terms/size/size.md
Comment (SSwiniarski) [inline content/java/concepts/array-list/terms/size/size.md:24]: We'd like to see a sentence describing what the example does to introduce the example.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@dakshdeepHERE I did a first review and left a couple of comments.

File: content/java/concepts/array-list/terms/size/size.md
Comment (dakshdeepHERE) [reply content/java/concepts/array-list/terms/size/size.md:24]: alright I will add now :)

**Comment by dakshdeepHERE** [comment]
@SSwiniarski  all the changes requested done please check the lines you asked me to add are correct or not in the end example.

File: content/java/concepts/array-list/terms/size/size.md
Comment (SSwiniarski) [inline content/java/concepts/array-list/terms/size/size.md:25]:
```suggestion
The following example creates a `ArrayList`, uses `.add()` to add to elements to the `ArrayList`, and in the end uses `.size()` to print the size of the `ArrayList`:
```

File: content/java/concepts/array-list/terms/size/size.md
Comment (SSwiniarski) [inline content/java/concepts/array-list/terms/size/size.md:52]:
```suggestion
Size of list: 3
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@dakshdeepHERE, I have a few more comments.

**Comment by dakshdeepHERE** [comment]
done @SSwiniarski

**Review by SSwiniarski — APPROVED** [review]
@dakshdeepHERE Looks good for a second review.

File: content/java/concepts/array-list/terms/size/size.md
Comment (yangc95) [inline content/java/concepts/array-list/terms/size/size.md:20]: Can we include a sentence about what is returned (can mention the type which would explain the explicit `int` and link out to https://www.codecademy.com/resources/docs/java/data-types). Also, include what `list` represents.

Comment (dakshdeepHERE) [reply content/java/concepts/array-list/terms/size/size.md:20]: sure and @yangc95 can you let me know why my old pr are not getting merged even after changing everything that I was asked to change?

File: content/java/concepts/array-list/terms/size/size.md
Comment (yangc95) [inline content/java/concepts/array-list/terms/size/size.md:19]: Removing "we"pronoun here

```suggestion
Here, a variable `size` of type [`int`](https://www.codecademy.com/resources/docs/java/data-types) is initialized with the size of a `list`. The `List` [interface](https://www.codecademy.com/resources/docs/java/interfaces) is an ordered collection of elements and is implemented by various classes such as `ArrayList`, `LinkedList`, and `Vector`:
```

File: content/java/concepts/array-list/terms/size/size.md
Comment (yangc95) [reply content/java/concepts/array-list/terms/size/size.md:22]: > can you let me know why my old pr are not getting merged even after changing everything that I was asked to change?

Thanks for the changes 🙂 Maintainers are working to get every submitted PR up to standards and merged in a timely manner so ty for your patience!

**Comment by dakshdeepHERE** [comment]
All done @yangc95 ? Also my old PRs are not getting merged even after all checks done is it because the author is not active?

**Comment by yangc95** [comment]
> All done... Also my old PRs are not getting merged even after all checks done is it because the author is not active?

@dakshdeepHERE for your PRs, you would be the author. But no need to stress over any unmerged PRs as the progression for each is case-by-case. They'll eventually be dealt with accordingly! Feel free to tag any maintainers on the PR thread in question if you have any concerns.

**Comment by dakshdeepHERE** [comment]
@yangc95 
image
I don't have the permission to initiate a merge after all the checks are done can you tell me what you meant when you said I'm the author.

**Comment by yangc95** [comment]
> can you tell me what you meant when you said I'm the author.

Yeah! When I said you're the author, I meant that when this PR is merged (because you created it), you'll appear as the contributor on this entry. The "Author" is displayed on every PR thread's comment next to username:
Screen Shot 2023-01-18 at 3 05 48 PM

Only maintainers are allowed to merge after an entry meets standards 🙂

File: content/java/concepts/array-list/terms/size/size.md
Comment (yangc95) [inline content/java/concepts/array-list/terms/size/size.md:27]:
```suggestion
The following example creates an `ArrayList`, uses [`.add()`](https://www.codecademy.com/resources/docs/java/array-list/add) to append elements to it, and uses `.size()` to print out its size:
```

**Review by yangc95 — COMMENTED** [review]
@dakshdeepHERE I only have one comment left. Afterwards I think we can merge!

cc: @SSwiniarski

File: content/java/concepts/array-list/terms/size/size.md
Comment (yangc95) [inline content/java/concepts/array-list/terms/size/size.md:53]:
```suggestion
```shell
```

**Review by yangc95 — APPROVED** [review]
Merging this PR!

I made a quick change to the output block here: https://github.com/Codecademy/docs/pull/1470/commits/4abd596a4c61d42e4d66f74619ca9642c1302e14

**Comment by yangc95** [comment]
🙌 @dakshdeepHERE live link: https://www.codecademy.com/resources/docs/java/array-list/size

Woohoo! All set:
Screen Shot 2023-01-27 at 6 26 35 PM

cc: @SSwiniarski @KTom101

**Pull request merged**
